.rubocop_todo.yml

```diff
@@ -1,57 +1,7 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2016-07-09 23:34:13 +0200 using RuboCop version 0.41.2.
+# on 2016-10-14 23:34:08 +0530 using RuboCop version 0.44.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
-
-# Offense count: 3
-Lint/AmbiguousRegexpLiteral:
```

Comment:
Any reason why you moved everything into  .rubocop.yml ? I guess the idea is to define in_ .rubocop.yml _ basic changes to the default values, whereas in  _.rubocop_todo.yml _ you keep exclusions which you are temporarily excluding until you can fix it. Would you agree? If so, are you saying the exclusions in _ .rubocop.yml _  now are a won't fix?